common/test/db_cache/bok_choy_schema_student_module_history.sql

```diff
@@ -34,7 +18,7 @@ CREATE TABLE `django_migrations` (
   `name` varchar(255) NOT NULL,
   `applied` datetime(6) NOT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=119 DEFAULT CHARSET=utf8;
+) ENGINE=InnoDB AUTO_INCREMENT=88 DEFAULT CHARSET=utf8;
```

Comment:
Is this something we introduced manually, or is it managed by an automated tool/process?  If we made the change ourselves I'm just interested in what the value represents and why we picked "88".